### DISCUSSION TITLE
Using a Proxy Host entry for NPMplus Web UI Issue

### DISCUSSION THREAD

**Discussion by nealhead** [ZoeyVid/NPMplus#943] [Q&A] [answered]
I tried setting up a proxy host to reach NPMplus by fqdn locally with a wildcart certificate is not working as intended as it did with NPM.

When I enter the URL, it attempts to redirect to port 81. If I append /login to the URL I can reach the login page but when I try to login it again tries to redirect to port 81.

It doesn't seem to be generating a log for this. 

Version: 2.11.2+454f3aa

**Comment by nealhead** [comment]
I may have solved this myself... by using Custom Location and setting to /nginx

**Comment by Zoey2936** [comment, marked as answer]
You need to use https as sheme

**Comment by gabriel-gfs** [comment]
![image](https://github.com/user-attachments/assets/a3ea43ea-c2d9-48a9-a876-2d82a126b41c)
I am still getting this issue where when going to https://nginx.domain.de i get redirected to https://nginx.domain.de:81/login which doesnt work.

**Reply by Zoey2936** [reply]
can you try it in a different browser? (so that there is no cache)

**Reply by gabriel-gfs** [reply]
Thanks. That appears to have done the trick
